src/renderer/gl_engine/tvgGlRenderer.cpp

```diff
@@ -112,6 +112,14 @@ bool GlRenderer::sync()
 
     mRenderPassStack.clear();
 
+    //Release disposed resources
+    if (mDisposeTextures.count > 0) {
+        for (auto i = mDisposeTextures.begin(); i < mDisposeTextures.end(); ++i) {
+            glDeleteTextures(1, i);
```

Comment:
it should be glDeleteTextures(1, &i) 
The parameter is GLuint*, see https://registry.khronos.org/OpenGL-Refpages/es2.0/xhtml/glDeleteTextures.xml